cypress/support/commands.js

```diff
@@ -0,0 +1,42 @@
+
+
+Cypress.Commands.add('loginFromUi', () => {
+    // login With UI
+     cy.visit('https://staging.circlepay.ai/')
+      cy.get('.mat-input-element').type('+923244323448').should('have.value','3244323448')
+      cy.contains('Get Started').click()
+      cy.url().should('include', '/auth')
+      cy.get('.password-field').type('Admin125!@%')
+      cy.contains('Get Started').click()
+     
+      cy.setCookie('CIRCLE_PAY_JWT_TOKEN', 'res.body.data.token')
```

Comment:
@AmnaShahid96 this step is not needed. Actually there is no response to save the token from its body 😅